src/app.ts

```diff
@@ -1,6 +1,72 @@
+type ResultData = {
+    id: string,
+    gender: string,
+    age: number,
+    row: number,
+}
+
+type PagingData = {
+    next?: string,
+    previous?: string,
```

Comment:
Codacy has a fix for the issue: [Replace `··previous?:·string,` with `previous?:·string;`](https://app.codacy.com/gh/dekunleroy/pipeline_assessment/pullRequest?prid=9962768)
```suggestion
  previous?: string;
```